src/chunkserver/datastore/file_pool.cpp

```diff
@@ -192,6 +195,14 @@ int FilePoolHelper::DecodeMetaInfoFromMetaFile(
             break;
         }
 
+        if (!value[kChunkNum].isNull()) {
```

Comment:
How to be compatible with meta files of older versions so that they can be upgraded